R/roll.R

```diff
@@ -0,0 +1,5 @@
+rollmean <- function(x, n, fill=NA, align=c("right", "left", "center"), na.rm=FALSE, adaptive=is.list(n)) {
+  align = match.arg(align)
+  ans = .Call(Crollmean, x, as.integer(n), fill, align, na.rm, adaptive)
```

Comment:
this `as.integer(n)` needs to be pushed to C to handle `is.list(n)`

Reply:
I'd have to look another time, but your statement sounds correct.